### DISCUSSION TITLE
Optional cosmetic plus (+) symbol next to the category names

### DISCUSSION THREAD

**Discussion by ArefinB** [JustOptimize/return-ShowHiddenChannels#157] [Ideas]
Could you add a small cosmetic feature where there will be plus (+) symbol next to the category names? This can be an option hidden in the plugin settings menu.

![image](https://github.com/JustOptimize/return-ShowHiddenChannels/assets/135141920/bf99be0f-cc98-43eb-bbca-69f71952c47f)

**Comment by JustOptimize** [comment]
The plus symbol means that you can create a channel in that category, why should you see that for hidden categories?
